From 7c4b256b2a8d0db103d7c7e38b591317d4b60fc6 Mon Sep 17 00:00:00 2001
From: Dmitry Dementyev <dementyev@google.com>
Date: Mon, 6 Mar 2023 21:34:13 +0000
Subject: [PATCH] Convert argument to Intent in car settings
 AddAccountActivity.

Bug: 265798353
Test: manual
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:2825e3b674a36f4c34fd504f53501010a621fcff)
Merged-In: I705fbc6929a20325d1a555c0bedf1853cd565933
Change-Id: I705fbc6929a20325d1a555c0bedf1853cd565933
---
 src/com/android/car/settings/accounts/AddAccountActivity.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/car/settings/accounts/AddAccountActivity.java b/src/com/android/car/settings/accounts/AddAccountActivity.java
index f166887cd..f497b31a9 100644
--- a/src/com/android/car/settings/accounts/AddAccountActivity.java
+++ b/src/com/android/car/settings/accounts/AddAccountActivity.java
@@ -95,7 +95,7 @@ public class AddAccountActivity extends Activity {
                     intent.putExtras(addAccountOptions);
                     intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                     startActivityForResultAsUser(
-                            intent, ADD_ACCOUNT_REQUEST, mUserHandle);
+                            new Intent(intent), ADD_ACCOUNT_REQUEST, mUserHandle);
                 } else {
                     setResult(RESULT_OK);
                     if (mPendingIntent != null) {
-- 
2.40.0.634.g4ca3ef3211-goog

